x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/selectors.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AlertListState } from './types';
+
+export const alertListData = (state: AlertListState) => state.alerts;
```

Comment:
consider naming this type `AlertingState` and this reducer `alertingReducer` and the naming the piece of global state `alerting`

Reply:
Will address this in the next PR where I add a stub alert detail